### PR TITLE
Feature/improving ai

### PR DESCRIPTION
**Pull request by Nom1fan**
Implemented getValidMoves for Pawn

### CODE REVIEW

File: quoridorcore/src/main/java/com/harush/zitoon/quoridor/core/model/Board.java [hunk 1]
```diff
@@ -108,7 +112,7 @@ public boolean containsWall(int x, int y, boolean isHorizontal) {
     }
 
     public boolean containsPawn(int x, int y) {
```
Comment (maorharush) [inline quoridorcore/src/main/java/com/harush/zitoon/quoridor/core/model/Board.java:115]: In Java, boolean properties' are by convention named with 'is'.
It's bad practice to break standard convention.

**Review by maorharush — COMMENTED** [review]
keep up the good job, you are entitled for a raise.
contact HR for info

File: quoridorcore/src/main/java/com/harush/zitoon/quoridor/core/model/Board.java (hunk 1 above)
Comment (Nom1fan) [reply quoridorcore/src/main/java/com/harush/zitoon/quoridor/core/model/Board.java:115]: Perhaps I wasn't clear enough. It's not specifically `is`. It should form a question:
[Click here for more info](https://stackoverflow.com/questions/3874350/naming-conventions-for-java-methods-that-return-booleanno-question-mark)

**Review by maorharush — COMMENTED** [review]
seems legit

**Review by maorharush — COMMENTED** [review]
looks good

**Review by maorharush — COMMENTED** [review]
looks much cleaner, why did we deleted to remvoeWall() API though?

**Pull request closed without merging**